e2e/tests/cohorts/cohort-group-actions.spec.ts

```diff
@@ -0,0 +1,253 @@
+import CohortActionsPage from 'app/page/cohort-actions-page';
+import CohortBuildPage from 'app/page/cohort-build-page';
+import WorkspaceDataPage from 'app/page/workspace-data-page';
+import { MenuOption, ResourceCard } from 'app/text-labels';
+import { makeRandomName, makeWorkspaceName } from 'utils/str-utils';
+import { findOrCreateWorkspace, signInWithAccessToken } from 'utils/test-utils';
+import { waitWhileLoading } from 'utils/waits-utils';
+import { getPropValue, waitUntilChanged } from 'utils/element-utils';
+import ClrIconLink from 'app/element/clr-icon-link';
+import { PhysicalMeasurementsCriteria } from 'app/page/cohort-participants-group';
+import { Ethnicity } from 'app/page/cohort-participants-group';
+
+describe('Build cohort page actions', () => {
```

Comment:
New test suite to cover testing of group and criteria snowman menu actions, like `rename group`, `edit criteria`, etc.